.github/workflows/cypress-update-snapshots.yml

```diff
@@ -0,0 +1,79 @@
+name: Cypress update snapshots
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      testname:
```

Comment:
We may want to consider allowing a user to enter a comma-separated list of test names to run since occasionally design changes will have broad effects on snapshots in multiple tests, and having to run this action multiple times could be tedious

Reply:
Yeah, I thought about that, but didn't want to add features I didn't need _right now_ －since that would be more stuff for me to test and debug 😆

Reply:
Totally fair - at least if we remove/edit concurrency group we can have multiple snapshot generation jobs running in parallel. I did quickly test this workflow (with suggested changes) and it does work as expected 👍🏼

Reply:
Fair enough 😆 

I'm happy with merging this as-is and leaving extending this to work with multiple test names to whoever next needs it / doesn't want to manually kick off multiple jobs